src/DssSpell.t.sol

```diff
@@ -30,6 +31,11 @@ contract DssSpellTest is DSTest, DSMath {
     // this needs to be updated
     uint256 constant SPELL_CREATED = 1606152054;
 
+    // Previous spell; supply if there is a need to test prior to its cast() function being called on mainnet.
+    address constant PREV_SPELL = address(0x8D602692eE4b5f0ec33A22fe6547822377FDCc4c);
```

Comment:
I also don't see a reason to make this an address instead of a `SpellLike`. The original pattern is even consistent with what we do for the system values.

Reply:
I was going for consistency with the MAINNET_SPELL, which is defined just above this.

Reply:
From week-to-week we'll be able to move the `MAINNET_SPELL` down a couple of lines, instead of having to scroll down half the file to find where `PREV_SPELL` is declared.

Reply:
This is true, I'm okay with leaving as is.

Reply:
I don't care enough to argue, so fair enough.